pkg/controller/seed/reconciler.go

```diff
@@ -103,10 +103,15 @@ func (kcr *kubeSystemReconciler) reconcile(ctx context.Context, logger logr.Logg
 		image.Tag = ptr.To[string](version.Get().GitVersion)
 	}
 
+	lakomConfig, err := yaml.JSONToYAML(kcr.serviceConfig.CosignPublicKeys.Raw)
+	if err != nil {
+		return fmt.Errorf("failed to convert lakom config from json to yaml, %w", err)
+	}
+
 	resources, err := getResources(
 		generatedSecrets[constants.SeedWebhookTLSSecretName].Name,
 		image.String(),
-		kcr.serviceConfig.CosignPublicKeys,
+		string(lakomConfig),
```

Comment:
You can directly pass this as `[]byte` and then use `.secret.data` instead of `.secret.stringData`
```suggestion
		lakomConfig,
```

Reply:
With cfb8298 the lakom config is now stored in ConfigMap instead of Secret, now `lakomConfig` should be of type string wherever possible.